js/oop/Palindrome.js

```diff
@@ -12,6 +12,49 @@ export class Palindrome {
   }
 
   // write your code here
+  reverse() {
+    const cleanedValue = this.value.toLowerCase().replace(/[^a-z0-9]/g, "");
+
+    const reversedValue = cleanedValue.split("").reverse().join("");
+
+    return cleanedValue === reversedValue;
+  }
+
+  loop() {
+    const cleanedValue = this.value.toLowerCase().replace(/[^a-z0-9]/g, "");
+
+    const length = cleanedValue.length;
+
+    for (let i = 0; i < length / 2; i++) {
+      if (cleanedValue[i] !== cleanedValue[length - 1 - i]) {
+        return false;
+      }
+    }
+
+    return true;
+  }
+
+  recursive() {
+    this.value = this.value.replace(/[^a-zA-Z]/g, "").toLowerCase();
+  }
```

Comment:
Please check my review at https://github.com/ngodingbang/rwid-git/pull/3#discussion_r1279672424.

Reply:
change method name reserve to checkPalindromeReverse, loop to checkPalindromeLoop, and recursive to checkPalindromeRecursive.

Reply:
Using "Palindrome" phrase on checkPalindromeReverse() and checkPalindromeLoop() is verbose because that class name already tells what it is about. I suggest you to change it into checkUsingReverse() and checkUsingLoop().